src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php

```diff
@@ -76,6 +94,9 @@ public static function register($mode = 0)
             $trace = debug_backtrace(true);
             $group = 'other';
 
+            $isWeak = DeprecationErrorHandler::MODE_WEAK === $mode ||
+                DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode && ($isVendor = self::inVendors($file));
```

Comment:
Here, I choose to put the assignment inline, to avoid this : 

```php
if (DeprecationErrorHandler::MODE_WEAK_VENDORS === $mode) {
    // let's compute the variable only when it is needed.
    $isVendor = self::inVendors($file);
}
// rest of the code
```